include/net/wifimgr_drv.h

```diff
@@ -81,6 +81,9 @@ struct wifi_drv_api{
 			new_station_t cb);
 	int (*stop_ap)(struct device *dev);
 	int (*del_station)(struct device *dev, u8_t *mac);
+	int (*hw_test)(struct device *dev, int ictx_id,
+		       char *t_buf, u32_t t_len, char *r_buf,
+		       u32_t *r_len)
```

Comment:
; is missing, which will cause a build warning.

[  7%] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/wifi/wifimgr.c.obj
In file included from /home/kelvin/work/uwp5661/zephyr/subsys/wifi/wifimgr.h:15:0,
                 from /home/kelvin/work/uwp5661/zephyr/subsys/wifi/wifimgr.c:19:
/home/kelvin/work/uwp5661/zephyr/include/net/wifimgr_drv.h:87:1: warning: no semicolon at end of struct or union
 };
 ^